modules/ossm-install-cert-manager.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-install-cert-manager_{context}"]
+== Install cert-manager
+
+To install cert-manager, follow these steps:
+
+//. Install cert-manager:
+//+
+//[source, terminal]
+//----
+//helm install cert-manager jetstack/cert-manager \
+//    --namespace cert-manager \
+//    --create-namespace \
+//    --version v1.11.0 \
+//    --set installCRDs=true
+//----
+
+. Provision certificates:
++
+[source, terminal]
+----
+oc new-project istio-system
+oc apply -f deploy/examples/cert-manager/selfsigned-ca.yaml -n istio-system
```

Comment:
Adding the Yaml will be clear, path will not clear for the Customers

Command:

`oc apply -f  <selfsigned-ca.yaml> -n istio-system
`
```
apiVersion: cert-manager.io/v1
kind: Issuer
metadata:
  name: selfsigned
spec:
  selfSigned: {}
---
apiVersion: cert-manager.io/v1
kind: Certificate
metadata:
  name: istio-ca
spec:
  isCA: true
  duration: 2160h # 90d
  secretName: istio-ca
  commonName: istio-ca
  subject:
    organizations:
      - cluster.local
      - cert-manager
  issuerRef:
    name: selfsigned
    kind: Issuer
    group: cert-manager.io
---
apiVersion: cert-manager.io/v1
kind: Issuer
metadata:
  name: istio-ca
spec:
  ca:
    secretName: istio-ca
```

Reply:
+1